charts/umbrella/values.yaml

```diff
@@ -188,3 +188,30 @@ discoveryfinder:
       initialDelaySeconds: 200
   postgresql:
     nameOverride: "discoveryfinder-postgresql"
+
+sdfactory:
```

Comment:
As  @evegufy  pointed out in a call `enabled` condition is still missing in the values.

Reply:
done